conf/application.conf

```diff
@@ -50,6 +50,7 @@ json.encryption = ${cookie.encryption}
 featureFlags {
   ivEnabled = false
   newRegistrationJourneyEnabled = true
+  agentJourney2026Enabled = true
 }
```

Comment:
There's two agentJourney2026Enabled here. I think this one can be removed and we can keep the one in feature-switch.